code/__HELPERS/atoms.dm

```diff
@@ -360,7 +360,7 @@ rough example of the "cone" made by the 3 dirs checked
  * This part of the los_check proc is meant to be overriden by subtypes for things they specially need to check or do.
  * args are always the same as passed in los_check, but with the turf being checked and the dummy obj passed at the end too.
  */
-/datum/proc/mid_los_check(atom/movable/user, mob/target, pass_args, turf/next_step, dummy)
+/atom/proc/mid_los_check(atom/movable/user, mob/target, pass_args, turf/next_step, dummy)
 	return TRUE
```

Comment:
This shouldn't exist - instead you should be able to pass a callback into ``/proc/los_check()`` which gets invoked as a check.